postcss.config.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
-	plugins: ['postcss-import', 'tailwindcss', 'autoprefixer'],
-};
+  plugins: [
+    // require('postcss-import'),
+    require('tailwindcss'),
```

Comment:
Hey @saintaze thanks for the contribution and for the heads up about the postcss import plugin! I've noticed that those changes related to the postcss plugins are breaking the build, they must be passed as strings because require doesn't work here https://github.com/vercel/next.js/blob/master/errors/postcss-shape.md#common-errors

Reply:
that is strange, I checked and it seems to give an error on the string syntax and works with require syntax locally at least.  I took the `require` syntax straight from Tailwind docs [tailwind config](https://tailwindcss.com/docs/using-with-preprocessors)